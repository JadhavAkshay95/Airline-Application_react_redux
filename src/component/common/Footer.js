import React from 'react';
import Container from 'react-bootstrap/Container';
import Row from 'react-bootstrap/Row';
import Col from 'react-bootstrap/Col';

var style = {
  backgroundColor: '#e6fff8',
  borderTop: '1px solid #E7E7E7',
  textAlign: 'center',
  padding: '20px',
  position: 'fixed',
  left: '0',
  bottom: '0',
  width: '100%'
};

const Footer = () => {
  return (
    <div style={style}>
      <Container>
        <Row>
          <Col></Col>
        </Row>
      </Container>
    </div>
  );
};

export default Footer;
