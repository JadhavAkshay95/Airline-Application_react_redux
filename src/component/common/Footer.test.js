import React from 'react'
import { shallow } from 'enzyme'
import Footer from './Footer'

describe('Render Footer component', () => {
  it('Render Footer ', () => {
    const wrapper = shallow(<Footer />)
    expect(wrapper).toMatchSnapshot()
  })
})
