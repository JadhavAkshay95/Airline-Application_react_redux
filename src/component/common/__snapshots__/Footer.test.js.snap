// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Footer component Render Footer  1`] = `ShallowWrapper {}`;
